core/egov-localization/src/main/java/org/egov/domain/service/MessageService.java

```diff
@@ -104,13 +107,50 @@ public void bustCache() {
         return messages;
     }
     
-    public boolean createMessage(CreateMessagesRequest createMessagesRequest){
-    	messageRepository.deleteMessages(createMessagesRequest);
-   		boolean createStatus = messageRepository.createMessage(createMessagesRequest);
+    public boolean createMessage(NewMessagesRequest newMessagesRequest){
+    	List<Message> messageList = getOnlyMessages(newMessagesRequest);
+    	messageRepository.deleteMessages(newMessagesRequest.getLocale(), newMessagesRequest.getTenantId(), messageList);
+   		boolean createStatus = messageRepository.createMessage(newMessagesRequest.getLocale(), newMessagesRequest.getTenantId(), messageList);
    		if(createStatus){
    			return true;
    		}
    		return false; 
     }
-
+    
+    public boolean deleteMessage(DeleteMessagesRequest deleteMessagesRequest){
```

Comment:
@darshannagesh  Similar to the above comment. In the controller convert the deleteMessagesRequest to a list of domain Message list and then invoke the service.